src/encoded/schemas/experiment_tsaseq.json

```diff
@@ -0,0 +1,117 @@
+{
+    "title": "TSA-seq Experiment",
+    "description": "Schema for TSA-seq experiments.",
+    "id": "/profiles/experiment_tsaseq.json",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "required": ["experiment_type", "award", "lab", "biosample"],
+    "identifyingProperties": ["uuid", "accession", "aliases"],
+    "additionalProperties": false,
+    "mixinProperties": [
+        { "$ref": "mixins.json#/schema_version" },
+        { "$ref": "mixins.json#/accession" },
+        { "$ref": "mixins.json#/uuid" },
+        { "$ref": "mixins.json#/aliases" },
+        { "$ref": "mixins.json#/attribution" },
+        { "$ref": "mixins.json#/submitted" },
+        { "$ref": "mixins.json#/modified" },
+        { "$ref": "mixins.json#/release_dates" },
+        { "$ref": "mixins.json#/notes" },
+        { "$ref": "mixins.json#/references" },
+        { "$ref": "mixins.json#/dbxrefs" },
+        { "$ref": "mixins.json#/documents" },
+        { "$ref": "mixins.json#/library"},
+        { "$ref": "mixins.json#/sop_mapping"},
+        { "$ref": "mixins.json#/tags" },
+        { "$ref": "mixins.json#/badges" },
+        { "$ref": "mixins.json#/antibody_info" },
+        { "$ref": "mixins.json#/spikein_info" },
+        { "$ref": "experiment.json#/properties" }
+    ],
+    "dependencies": {},
+    "mixinFacets": [
+        { "$ref": "experiment.json#/facets"},
+        { "$ref": "mixins.json#/facets_common" }
+    ],
+    "mixinColumns": [
+        { "$ref": "experiment.json#/columns"}
+    ],
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "experiment_type": {
```

Comment:
Since this is a TSA-seq-only schema, possibly edit experiment_type to include "default": "TSA-seq" and exclude from Submit4dn and FFedit-create